docs-new/docs/graphql/core/actions/types.mdx

```diff
@@ -117,22 +114,14 @@ type UserInfo {
 }
 ```
 
-<div class="note">
-
-<div class="title">
-
-Note
-
-</div>
+:::info Note
 
 Only fields with non-list scalar types (e.g. `Int`, `String`) can be
 used to define relationships
 
-</div>
+:::
 
-<div class="admonition">
-
-Limitations
+:::info Limitations
```

Comment:
can we change this to `:::caution Limitations` to get the message go with color - more visual info